docs/roi_tables.md

```diff
@@ -0,0 +1,333 @@
+4. We store ROIs associated to segmented objects, for instance the bounding boxes of organoids/nuclei.
+5. We store measurements associated to segmented objects (e.g. as computed via `regionprops` from `scikit-image`, as wrapped in [napari-skimage-regionprops](https://github.com/haesleinhuepf/napari-skimage-regionprops)). Note: we store these tables with the the same AnnData format as described in this page, but they are not *ROI* tables and therefore they are not based on the current specifications (notably with respect to the [required columns](#table-contents)).
+
+
+## Table specifications
+
+The current section describes the first version (V1) of `fractal-tasks-core`
+tables, which is based on [a proposed update to NGFF
+specs](https://github.com/ome/ngff/pull/64); this update is currently on hold,
+and `fractal-tasks-core` will evolve as soon as the NGFF specs will adopt a
+definition of tables.
+As in the original proposed NGFF update, the current specifications are
+specifically based on AnnData tables.
+
+### Zarr structure
+
+The structure of Zarr groups is based on the [`image` specification in NGFF 0.4](https://ngff.openmicroscopy.org/0.4/index.html#image-layout), with an additional `tables` group and the corresponding subgroups (similar to `labels`):
+```
+image.zarr        # Zarr group for a NGFF image
+|
+├── 0             # Zarr array for multiscale level 0
+├── ...
+├── N             # Zarr array for multiscale level N
+|
+├── labels        # Zarr subgroup with a list of labels associated to this image
+|   ├── label_A   # Zarr subgroup for a given label
+|   ├── label_B   # Zarr subgroup for a given label
+|   └── ...
+|
+└── tables        # Zarr subgroup with a list of tables associated to this image
+    ├── table_1   # Zarr subgroup for a given table
+    ├── table_2   # Zarr subgroup for a given table
+    └── ...
+```
+
+### Zarr attributes
+
+#### Tables container
+
+The Zarr attributes of the `tables` group must include the key `tables`,
+pointing to the list of all tables; this simplifies the discovery of image
+tables.
+
+Here is an example of `image.zarr/tables/.zattrs`:
+```json
+{
+    "tables": ["table_1", "table_2"]
+}
+```
+
+#### Single table (default)
+
+For each table, the Zarr attributes must include the key
+`fractal_roi_table_version`, pointing to the string version of this
+specification (e.g. `1`).
+
+Here is an example of `image.zarr/tables/table1/.zattrs`
+```json
+{
+    "fractal_roi_table_version": "1",
+    "encoding-type": "anndata",      # Automatically added by anndata
+    "encoding-version": "0.1.0",     # Automatically added by anndata
+}
+```
+
+This is the kind of tables that are used in `fractal-tasks-core` to store ROIs
+like a whole well, or the list of field of views.
+
+#### Single table (segmented objects)
+
+When each table row corresponds to (the bounding box of) a segmented object,
```

Comment:
I'm confused now. Aren't the ROI tables the bounding box tables? Is this a feature table (as e.g. coming from napari workflow wrapper?) or a masked ROI table (e.g. cellpose table output?